ci/buildspec_gpu.yml

```diff
@@ -7,9 +7,8 @@ phases:
     commands:
       - apt-get update
       - apt-get install sudo -y
-      - python ts_scripts/install_dependencies.py --cuda=cu102 --environment=dev
+      - pip install -r ci/launcher/requirements.txt
 
   build:
     commands:
-      - python torchserve_sanity.py
-      - cd serving-sdk/ && mvn clean install -q && cd ../
\ No newline at end of file
+      - python ci/launcher/launch_test.py --instance-type p3.2xlarge
```

Comment:
Can you keep the old buildspec_gpu.yml/buildspec_cpu.yml and create the corresponding new ones so that we can easily switch?